.github/workflows/pr-title-check.yml

```diff
@@ -0,0 +1,22 @@
+name: PR Title Check
+
+on:
+  pull_request:
+    types: [opened, edited, synchronize]
```

Comment:
we can run it just with opened pr to avoid unnecessary triggers

Reply:
https://github.com/juntossomosmais/atomium/pull/526/commits/537e8a3595e437a67a3e1da0c409343e0712c143